### PR TITLE
Capstone Project: Online Learning Platform

### PR DESCRIPTION
**Pull request by AFakhriddinov**
**In this milestone,**

Opened a new repository
Created a new website for capstone project
Website was made responsive for desktop and mobile screens
Middle section data is taken from JavaScript file dynamically
Included mobile menu with toggle button
Link to previous PR: https://github.com/AFakhriddinov/English-learning-Course/pull/1
Link to [demo presentation](https://screenrec.com/share/rRVA1lTg86)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
```
Comment (arslanbisharat) [inline .vscode/settings.json]: - [x] It is recommended to not have .vscode in your project as it makes your project unclean.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">JavaScript</a></li>
+    <li><a href="#">GitHub</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+
```
Comment (arslanbisharat) [inline README.md:52]: - [x] you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

A description of the project.
A demo of the project features (different pages and different screen sizes).
You should also highlight some interesting piece of code or something you built that you are very proud of.
For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture.

Add the video link to your pull request description.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <!-- <script
+      src="https://kit.fontawesome.com/a076d05399.js"
+      crossorigin="anonymous"
+    ></script> -->
```
Comment (arslanbisharat) [inline about.html]: - [x] Let's remove commented code as it makes code hard to read.

File: about.html [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+
+    <link rel="stylesheet" href="./styles.css" />
+
+    <title>English Learning Course</title>
+  </head>
+  <body>
+    <!-- Navigation section -->
+    <div class="main-top-header">
```
Comment (arslanbisharat) [inline about.html:18]: - [x] Please make sure It's possible to navigate between the pages, there's a link from Home page to About page and the opposite too.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
```
Comment (arslanbisharat) [inline about.html:1]: - [x] It is great to see you used your own design but please note you have to use the same design, color, layout, and fonts given by the university.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <!-- <script
```
Comment (arslanbisharat) [inline index.html]: - [x] Let's remove commented code.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @contributor_name, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi @AFakhriddinov  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**